feat: add README (#62)
revanced/revanced-manager: README.md

# ReVanced Manager
💊 **The official ReVanced Manager**
## What is ReVanced Manager?

> **ReVanced Manager is a GUI app that will let you build and patch other apps such as YouTube, YouTube Music, Reddit, Twitter, TikTok and more!**

Reviewer: ```suggestion
**ReVanced Manager is an Android app that will let you patch other apps such as YouTube, YouTube Music, Reddit, Twitter, TikTok, and more!**
```

> > **Our goal is to make it as easy and friendly as possible so everyone can use it :D**

Reviewer: ```suggestion
```

## Information

### Status

> 🟡 ReVanced Manager is currently in WiP (Work in Progress), which means that it's not ready yet to be used by everyone as we're still coding the app and adding plenty of things.

Reviewer 2: ```suggestion
🟡 ReVanced Manager is currently WIP (Work in Progress), usage of the app is not recommended as it is not complete and is under active development. 
```

#### ❔ Should I try building it?
> At the moment, we don't recommend normal users to build it as the code is quite unstable and we're adding new things by the minute, so it's always changing. The only exception, of course, is if you really want to help us and contribute to the project, in which we'd be really grateful!

> 
> **Normal users, please hang on tight and wait until the Manager is fully released! I promise you won't be disappointed ;)**

Reviewer: ```suggestion
```
